OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1519,10 +1519,6 @@
 				);
 				FRAMEWORK_VERSION = A;
 				GCC_ENABLE_OBJC_EXCEPTIONS = YES;
-				GCC_PREPROCESSOR_DEFINITIONS = (
-					"DEBUG=1",
-					"$(inherited)",
-				);
```

Comment:
These were added at the target level, and caused `DEBUG=1` to end up in the resolved macros twice.